projects/1.21/assets/ifr25ku/miragefairy2024/lang/zh_cn.json

```
@@ -0,0 +1,1094 @@
+{
+  "advancements.miragefairy2024.aura_reflector_furnace.description": "建造一个灵气反应熔炉，并向其中添加灵魂燃料。",
+  "advancements.miragefairy2024.aura_reflector_furnace.title": "自我聚合",
+  "advancements.miragefairy2024.bag.description": "制作一个能存放一定数量物品的袋子",
```

```suggestion
  "advancements.miragefairy2024.bag.description": "合成一个能收纳特定物品的包",
```
首先贴图很明显是包不是袋子，日文也是`カバン`
其次，英文用了`Craft`所以建议动词也用术语`合成`
然后语义也是错翻